fiftyone/utils/sam3.py

```
@@ -0,0 +1,537 @@
+        predictor = _SAM3Predictor(model=self._model)
+        tracker_model = predictor.processor.model
+        if getattr(tracker_model, "backbone", None) is None:
+            tracker_model.backbone = self._model.backbone
+        return predictor
+
+    def _load_model(self, config):
+        if "device" not in config.entrypoint_args:
+            config.entrypoint_args["device"] = self._device
+        return super()._load_model(config)
+
+    def _download_model(self, config):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_download_model`

Add return type annotation: `None`

(ANN202)